servicebox-app/src/main/java/io/halkyon/model/Service.java

```diff
@@ -0,0 +1,43 @@
+package io.halkyon.model;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import io.quarkus.panache.common.Sort;
+import org.jboss.resteasy.annotations.jaxrs.FormParam;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import java.util.List;
+
+@Entity
+public class Service extends PanacheEntityBase {
+
+    @Id
+    @SequenceGenerator(name = "svcSeq", sequenceName = "svc_id_seq", allocationSize = 1, initialValue = 5)
+    @GeneratedValue(generator = "svcSeq")
+    Long id;
+
+    @FormParam
+    public String name;
+
+    @FormParam
+    public String version;
+
+    /* in form of tcp:8080*/
+    @FormParam
+    public String endpoint;
+
+    @FormParam
+    public Boolean deployed;
```

Comment:
Can you also add a `created` java SQL date field (see Claim Model class + import SQL + add method where a defauld date time is added)  ?